core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -42,6 +42,7 @@ object TransactionManager {
   // default transaction management config values
   val DefaultTransactionalIdExpirationMs = TimeUnit.DAYS.toMillis(7).toInt
   val DefaultTransactionsMaxTimeoutMs = TimeUnit.MINUTES.toMillis(15).toInt
+  val DefaultRemoveExpiredTransactionsIntervalMs = TimeUnit.MINUTES.toMillis(1).toInt
```

Comment:
When do things need to go into `KafkaConfig`? If the motivation is to make this externally configurable, then I disagree: I don't think that this should be made externally configurable, how would users meaningfully set this?